ethers-ext/src/v6/gasless.ts

```diff
@@ -52,8 +50,23 @@ export function getAmountRepay(approveRequired: boolean, gasPrice: number = 25):
  * @param chainId The chain ID
  * @returns The gasless swap router contract
  */
-export function getGaslessSwapRouter(provider: ethers.Provider, chainId: number): any {
-  const routerAddress = CHAIN_ROUTER_ADDRESSES[validateChainId(chainId)];
+export async function getGaslessSwapRouter(provider: ethers.Provider, chainId: number): Promise<any> {
+  validateChainId(chainId)
+  assert(
```

Comment:
@Mdaiki0730 can you use SystemRegistry ABI + contract call instead of kaia RPC call? It would be better if we stick to pure Ethereum compatibility.

Reply:
@blukat29 I was wondering about this method, but I agree with you. Thank you.

Reply:
It have been fixed at 91a6b33.